rtv/page.py

```diff
@@ -226,17 +226,43 @@ def downvote(self):
         except praw.errors.LoginOrScopeRequired:
             show_notification(self.stdscr, ['Login to vote'])
 
-    def prompt_input(self, prompt):
+    @BaseController.register('+')
+    def login(self):
+        username = self.prompt_input('Enter username: ')
+        password = self.prompt_input('Enter password: ', hide=True)
+
+        if (username == '') or (password == ''):
+            curses.flash()
+            return
+        try:
+            self.reddit.login(username, password)
+            
+        except praw.errors.InvalidUserPass:
```

Comment:
Is this thrown only when the password is wrong, or if either the username doesn't exist or password is wrong?